coc/http.py

```diff
@@ -403,9 +403,6 @@ def get_location_players(self, location_id, **kwargs):
     def get_location_clans_versus(self, location_id, **kwargs):
         return self.request(Route("GET", "/locations/{}/rankings/clans-versus".format(location_id), **kwargs))
 
-    def get_location_clans_capital(self, location_id, **kwargs):
-        return self.request(Route("GET", "/locations/{}/rankings/capitals".format(location_id), **kwargs))
-
```

Comment:
This also got added with the capital leaderboards, so it should disappear once you merged them in.